content/en/blog/_posts/2022-12-08-kubernetes-1.26-blog.md

```diff
@@ -0,0 +1,208 @@
+
+Introduced in Kubernetes v1.24, [this feature](https://github.com/kubernetes/enhancements/issues/3031) constitutes a significant milestone in improving the security of the Kubernetes release process. All release artifacts are signed keyless using [cosign](https://github.com/sigstore/cosign/), and both binary artifacts and images [can be verified](https://kubernetes.io/docs/tasks/administer-cluster/verify-signed-artifacts/).
+
+### Support for Windows privileged containers graduates to stable
+
+Privileged container support allows containers to run with similar access to the host as processes that run on the host directly. Support for this feature in Windows nodes, called HostProcess containers, will now [graduate to Stable](https://github.com/kubernetes/enhancements/issues/1981), enabling access to host resources (including network resources) from privileged containers.
+
+### Improvements to Kubernetes metrics
+
+This release has several noteworthy improvements on metrics.
+
+#### Metrics framework extension graduates to alpha
+
+The metrics framework extension [graduates to Alpha](https://github.com/kubernetes/enhancements/issues/3498), and [documentation](/docs/reference/instrumentation/metrics/) is now published for every metric in the Kubernetes codebase. This enhancement adds two additional metadata fields to Kubernetes metrics: `Internal` and `Beta`, representing different stages of metric maturity.
+
+#### Component Health Service Level Indicators graduates to alpha
+
+Also improving on the ability to consume Kubernetes metrics, [component health Service Level Indicators (SLIs)](https://github.com/kubernetes/enhancements/issues/3466) have [graduated to Alpha](https://github.com/kubernetes/kubernetes/pull/112884): by enabling the `ComponentSLIs` feature flag there will be an additional metrics endpoint which allows the calculation of Service Level Objectives (SLOs) from raw healthcheck data converted into metric format.
+
+#### Feature metrics are now available
+
+Feature metrics are now available for each Kubernetes component, making it possible to [track whether each active feature gate is enabled](https://github.com/kubernetes/kubernetes/pull/112690) by checking the component's metric endpoint for `kubernetes_feature_enabled`.
+
+### Dynamic Resource Allocation graduates to alpha
+
+[Dynamic Resource Allocation](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3063-dynamic-resource-allocation/README.md) is a new feature that puts resource scheduling in the hands of third-party developers: it offers an alternative to the limited "countable" interface for requesting access to resources (e.g. `nvidia.com/gpu: 2`), providing an API more akin to that of persistent volumes. Under the hood, it uses the [Container Device Interface](https://github.com/container-orchestrated-devices/container-device-interface) (CDI) to do its device injection. This feature is blocked by the `DynamicResourceAllocation` feature gate.
```

Comment:
Does it make more sense to link to the upcoming documentation for this feature than the KEP?

Reply:
We're using both the KEPs and the issues throughout the article (even because in some cases documentation can still be wip), do you think in this case it's really needed to change it? and is the documentation URL for this "stable"?

Reply:
I do think that people reading the article will expect to see links to documentation rather than to design documents or enhancements trackers.
SIG Docs adds redirects when we reorganize the website, so links should stay stable or at least continue to work even if we restructure things.

The blog post will go out _after_ the new docs are live. If not, we're doing it wrong.

Reply:
Related: it'd be nice to edit the issue description of graduated enhancements to include a link to the relevant docs. But we don't do this yet.

Reply:
I believe the documentation should be a stable link. This was the PR where it was merged: https://github.com/kubernetes/website/pull/37766

I'll let you guys be the judge as to which is more appropriate -- the KEP is also linked from the documentation, for reference.

 Independently, will the blog post for this feature also be linked here somewhere? From what I understand having a blog post ready was a pre-requisite for being included in the list for this blog post.

Reply:
You **don't** have to write a post-release blog article to have your feature mentioned in either:
- the release notes
- the release announcement.

You do have to have that article ready on time to be included in the _post release blog article series_.

Reply:
I can take a look at adding docs (I've based my work on previous examples and it wasn't common, hence the omission). In this case, if I wanted to add the URL (and eventually to the Graduated ones), it would have to be added during the short period after the docs are updated (and the URL is live) and this one is released - my only concern here is pushing something that could open more discussion (change the link description, use a different link word, etc) so close to the publication date.

Reply:
You can add the links now. They're on the `dev-1.26` branch and there's a preview at https://deploy-preview-37810--kubernetes-io-main-staging.netlify.app/

If docs aren't done yet (on that branch), don't link to them. No worries there. For this release in particular it feels to me that we are struggling to get the docs ready in time.

Reply:
https://github.com/kubernetes/website/pull/38123#discussion_r1037973299 describes a nice to have and is unimportant.

Reply:
Thank you, will have a look and make some consistent changes during the weekend - will try to be limited in them.

Reply:
@sftim @klueska could you take a look? I opted for using the docs to the term, and the link to the KEP when the "new feature" is mentioned.

Reply:
> I opted for using the docs to the term, and the link to the KEP when the "new feature" is mentioned.

That sounds good to me.

Reply:
I've added more links, using a similar approach, to other additions.